svthalia/concrexit
# Improve GSuite MailingList synchronisation

### Summary
Improve GSuite MailingList synchronisation

### How to test
Steps to test the changes you made:
1. Make the right configuration for mailinglists (it may be possible that review environments do not support this)
2. Check that the mailing lists that are part of concrexit get properly created/archived/updated
